src/tests/system/tests/test_feature.py

```diff
@@ -75,12 +75,14 @@ def test_feature__presence(
     if sssd_version["major"] > sssd_major or (
         sssd_version["major"] == sssd_major and sssd_version["minor"] >= sssd_minor
     ):
+        state = "" if presence else "not"
         assert client.features[feature] == presence, (
-            f"Feature {feature} should be present in {distribution} {distro_major}.{distro_minor} with "
-            "sssd-{sssd_major}.{sssd_minor}"
+            f"Feature {feature} should {state} be present in {distribution} {distro_major}.{distro_minor} with "
+            f"sssd-{sssd_version['major']}.{sssd_version['minor']}"
         )
     else:
+        state = "not" if presence else ""
         assert client.features[feature] != presence, (
-            f"Feature {feature} should not be present in {distribution} {distro_major}.{distro_minor} with "
-            "sssd-{sssd_major}.{sssd_minor}"
+            f"Feature {feature} should {state} be present in {distribution} {distro_major}.{distro_minor} with "
```

Comment:
Would it be possible to only set a var withint if-else and then assert outside? To avoid copy&paste.

Reply:
I don't think so. The assert and the output message depend on the sssd version, the expected feature presence (`presence`) and the actual feature presence (`client.features[feature]`). Even if I were to move the assert outside of the if-else condition I'd still have a similar logic inside it, which would involve the copy&paste that you mention.

Reply:
I mean something like:
```
    if sssd_version["major"] > sssd_major or (
        sssd_version["major"] == sssd_major and sssd_version["minor"] >= sssd_minor
    ):
        state = "" if presence else "not"
        expected = presence
    else:
        state = "not" if presence else ""
        expected = !presence

    assert client.features[feature] == expected, (
        f"Feature {feature} should {state} be present in {distribution} {distro_major}.{distro_minor} with "
        f"sssd-{sssd_version['major']}.{sssd_version['minor']}"
    )
```

Reply:
Good code! I adapted it a little bit ;)

Reply:
Thank you for the update.